### PR TITLE
Hacker Gloves

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]:

:cl: Hacker Gloves
add: Hacker gloves. Makes your hands work as multitools on wires, are also insulated. I'll probably figure out how to add them to the CE's inventory or something.
/:cl:

[why]: Was suggested in accepted suggestions. 
huge kudos to Carbonhell for his help

image

### CODE REVIEW

**Comment by ghost** [comment]
It is bro but once you get it you'll be able to do it quick as hell... Travis failed let's see why

**Comment by ghost** [comment]
You have an indentation error, if you click on the "Details" link on travis it'll take you to the attempted builds. You just need to change hackergloves.dm to be evenly spaced and your PR should be running proper.

**Comment by Carbonhell** [comment]
This will only work on doors! But pmuch anything else requiring a multitool will fail since the checks check for inhand,not gloves. Not sure if it is the behaviour you want

**Comment by ghost** [comment]
DM compiler version 512.1403
loading hippiestation.mdme
hippiestation/code/game/objects/items/hackergloves.dm:19:error: inconsistent indentation
hippiestation/code/game/objects/items/hackergloves.dm:27:error: inconsistent indentation
hippiestation.mdme.dmb - 2 errors, 0 warnings (2/19/18 6:19 am)
mv: cannot stat ‘hippiestation.mdme.dmb’: No such file or directory
mv: cannot stat ‘hippiestation.mdme.rsc’: No such file or directory


``
/datum/wires/ui_act(action, params) //enormous thanks to Carbonhell for helping enormously with this
    if("pulse")
        var/mob/living/user = usr 
        if(istype(user.slot_gloves, /obj/item/clothing/gloves/color/hacker))
``

``
if(..())
    . = TRUE//So we dont end up pulsing twice if we have both gloves and multitool.
``

**Comment by ghost** [comment]
Wow I formatted that comment terribly. Point being fix the indent errors on line 19 and 27

**Comment by ghost** [comment]
ill have to redo it anyways,  possibly have to also eddit wires.dmi

**Comment by ghost** [comment]
can i just add a check to wires.dmi, @Carbonhell?  or is that snowflake

**Comment by ghost** [comment]
dm*

**Comment by ghost** [comment]
That does sound a little snowflakey.. IDK, but the more disconnected the code gets the harder it is for someone new to walk in and modify it.

**Comment by FantasticFwoosh** [comment]
Petty thieving assistant puts their hacker-man insulated gloves to use.

I love the powerglove, its so bad.

**Comment by Carbonhell** [comment]
Wrong way to approach the problem. I would prefer if you made the tool quality check mob proc to check for gloves too, and then just give your gloves the MULTITOOL flag.

**Comment by Carbonhell** [comment]
The idea got my +1 anyway, next time try to use suggestion forum first tho

**Comment by ghost** [comment]
if it works and is tested, yeah is pretty cool +1

**Comment by Pyko1** [comment]
+1

File: hippiestation/code/game/objects/items/hackergloves.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	siemens_coefficient = 0
+	permeability_coefficient = 0.05
+	item_color="yellow"
+	resistance_flags = NONE
+
+/datum/wires/ui_act(action, params) //enormous thanks to Carbonhell for helping enormously with this
+
+        if("pulse")
+
+                var/mob/living/user = usr
```
Comment (tortellinitony) [inline hippiestation/code/game/objects/items/hackergloves.dm:15]: fix all of this indentation

File: hippiestation/code/modules/jobs/job_types/engineering.dm [hunk 2]
```diff
@@ -1,2 +1,2 @@
 /datum/outfit/job/ce
-  backpack_contents = list(/obj/item/melee/baton/stungun=1, /obj/item/device/modular_computer/tablet/preset/advanced=1)
+  backpack_contents = list(/obj/item/melee/baton/stungun=1, /obj/item/device/modular_computer/tablet/preset/advanced=1, /obj/item/clothing/gloves/color/hacker=1)
```
Comment (tortellinitony) [inline hippiestation/code/modules/jobs/job_types/engineering.dm:2]: Space this list out into multiple lines to avoid conflicts

**Review by tortellinitony — CHANGES_REQUESTED** [review]

**Comment by praisenarsie** [comment]
stale ass bread, ping a maintainer for reopen

**Pull request closed without merging**